part-2-server/part-2/server.js

```diff
@@ -0,0 +1,32 @@
+var http = require ("http");
+var fs = require ("fs");
+
+var server = http.createServer(function(request, response) {
+	var url = request.url;
+
+	var numbers = []
+	for (var i = 0; i < 140; i ++) { 
+		numbers.push([i])
+	}
+	
+	if (url === "/poem-" + [i]) {
+		fs.readFile("/poem-" + [i] + ".txt", function(err, data) {
+			response.write(data.toString());
+			response.end();
+	});
+	} else if (url === "/") {
+		fs.readFile("index.html", function(err, data) {
+			response.write(data.toString());
+			response.end();
+	});
+	} else if (url === "/page-2") {
```

Comment:
Missing the 'page-3' coniditional, should response with page-2.html if the response,url is /page-2 OR /page-3